VultisigApp/VultisigApp/Localizables/en.lproj/Localizable.strings

```diff
@@ -363,7 +363,7 @@
 "somethingWentWrongTryAgain" = "Something went wrong, please try again.";
 "newUpdateAvailable" = "New Update Available";
 "appUpToDate" = "Application is up to date";
-"fastSign" = "Fast";
+"fastSign" = "Fast Sign";
```

Comment:
can you please create a new one for "fast Sign" , this one has also been used on the label 
<img width="1311" alt="image" src="https://github.com/user-attachments/assets/2c83503f-a7c8-4607-8f32-6a0823531038">